packages/node/src/login/oidc/OidcLoginHandler.ts

```diff
@@ -84,6 +85,35 @@ export default class OidcLoginHandler implements ILoginHandler {
         )}`
       );
     }
-    throw new NotImplementedError("getClient not implemented for Node");
+
+    const issuerConfig = await this.issuerConfigFetcher.fetchConfig(
+      options.oidcIssuer
+    );
+    let clientInfo: IClient;
+    if (options.clientId !== undefined && options.clientId !== undefined) {
+      clientInfo = {
+        clientId: options.clientId,
+        clientSecret: options.clientSecret,
+      };
+    } else {
+      // If client id and secret aren't specified in the options, performe dynamic
```

Comment:
Since these are OIDC spec-defined values, I'd be explicit in the comment. But again, are you being too implementation-specific here (and missing `client_name` too), e.g. we could just say 'the OIDC standard client registration details' or something instead?
```suggestion
      // If 'client_id' and 'client_secret' aren't specified in the options, perform dynamic
```

Reply:
The `client_name` is left out on purpose: only the client id and secret are required in the process, the client name just being a "nice to have". I find it more explicit to specifically mention client id and secret in the comment, rather than just ' OIDC standard client registration details', because this behaviour is spec-driven, so I wouldn't regard it as an implementation detail.

Reply:
Fixed comment in 723613e09a9a80001374bf3a8be9f5c37cb62ad1